Source/ACE.Server/Entity/GeneratorProfile.cs

```diff
@@ -391,7 +391,7 @@ public bool VerifyLandblock(WorldObject obj)
 
         public bool VerifyWalkableSlope(WorldObject obj)
         {
-            if (!obj.Location.Indoors && !obj.Location.IsWalkable())
+            if (!obj.Location.Indoors && !obj.Location.IsWalkable() && (obj.GravityStatus ?? false))
```

Comment:
PropertyBool.GravityStatus defaults to True for PhysicsObjs, as defined by the ac physics engine:

        public static readonly PhysicsState DefaultState =
            PhysicsState.EdgeSlide | PhysicsState.LightingOn | PhysicsState.Gravity | PhysicsState.ReportCollisions;